packages/xo-web/src/common/tooltip/index.js

```diff
@@ -115,14 +115,26 @@ export default class Tooltip extends Component {
     this._node = null
   }
 
-  _showTooltip = () => {
+  _showTooltip = event => {
     const { props } = this
+    const { position } = getPosition(
+      event,
+      event.currentTarget,
+      ReactDOM.findDOMNode(instance),
+      instance.state.place,
+      'solid',
+      {}
+    )
 
     instance.setState({
       className: props.className,
       content: props.content,
       show: true,
-      style: props.style,
+      style: {
+        ...props.style,
+        left: `${position.left}px`,
+        top: `${position.top}px`,
+      },
```

Comment:
Can you explain this change?

Reply:
In Chrome, the tooltip appears at the top of the page because the position is not defined.

Reply:
But why is it suddenly broken? Doesn't it work for every other tooltip?

Reply:
We add a tooltip on the disabled button for the first time. 
In Chrome, the "mousemove" event is not triggered on the disabled button.
So, we don't get the position.

Reply:
You should be able to remove this by rebasing on master: https://github.com/vatesfr/xen-orchestra/pull/5084.